README.md

```diff
@@ -1,31 +1,88 @@
 # nva-search-api
 A common search API for NVA across resources
 
+**Parameters to GET /search/resources**
 
-Structure wanted in response from the search API
+|Parameter|Description|Default value |
+|-----|-----|-----|
+| query | Term to search for in resource. | * 
+| orderBy | Field to order result by. |  modifiedDate 
+| sortOrder | Order of search results (**asc**ending or **desc**ending). | desc
+| from | Start posision for results, 0-based. | 0
+| results | Maximum number of publications in response. | 10
```

Comment:
Is there a max results? I.e. 10 (max 100)?

Reply:
Max results in this particular response, elasticSearch uses int as type for both from and results - for pagination. If one is interesteded in emptying the search database '_scroll' comes in handy....